components/grade-form.js

```diff
@@ -0,0 +1,22 @@
+class GradeForm {
+    constructor(formElement) {
+        this.formElement = formElement;
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.formElement.addEventListener('submit', this.handleSubmit)
+        this.createGrade = null;
+    }
+    onSubmit(createGrade) {
```

Comment:
This method's name doesn't seem to match the intended functionality. Maybe something like `assignCreateGradeCallback`?